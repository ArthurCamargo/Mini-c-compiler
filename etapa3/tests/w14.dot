digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n2;
n2 [label="10"];
n1 [label="return"];
n0 [label="f"];

}