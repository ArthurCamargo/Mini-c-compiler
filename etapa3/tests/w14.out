0x55d48fbe2350
0x55d48fbe2350, 0x55d48fbe2410
0x55d48fbe2410
0x55d48fbe2410, 0x55d48fbe23c0
0x55d48fbe23c0
0x55d48fbe23c0 [label="10"];
0x55d48fbe2410 [label="return"];
0x55d48fbe2350 [label="f"];
